docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,136 @@
+   microk8s.stop && microk8s.start && microk8s.status --wait-ready
+    ```
+
+    Further information on this subject can be [found here](https://github.com/ubuntu/microk8s/issues/493#issuecomment-498167435).
+
+3. The plugins are now enabled and the cluster bootstrapped. However, we must still wait for some microk8s internals (dns, http, and ingress) plugins to be ready. Failing to do so can result in pods entering a `CrashLoopBackoff` state:
+
+   ```bash
+    microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=kube-dns
+
+    microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=hostpath-provisioner
+
+    microk8s.kubectl wait -n ingress --for=condition=Ready pod -l name=nginx-ingress-microk8s
+    ```
+
+After these commands return successfully, we have ensured that dns, http, and nginx ingress are enabled and working properly inside the cluster.
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
+
+First we must install `helm` onto the cluster. With `helm`, we can easily install charts and manage the lifecycle of the installed artifacts. One example is the ability to rollback a failed install.
+
+1. __Codacy supports up to `helm` version 2.16.3.__
+   __We currently do not support `helm` v3.__
+2. Follow the instructions on installing the `helm` client on the [Preparing to install Codacy documentation](../../docs/index.md).
+
+3. We must now deploy `tiller` onto the cluster.
+   1. First we must set a `serviceaccount` and a `clusterrolebinding` for `tiller` on the microk8s cluster:
+
+      ```bash
+      microk8s.kubectl create serviceaccount --namespace kube-system tiller
+      microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+      ```
+
+   2. Then we can install `tiller`, which will be installed with the same `HELM_VERSION` that you have specified earlier and we wait for tiller to be ready:
+
+      ```bash
+      helm init --service-account tiller
+      microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller
+      ```
+
+### 2. Codacy chart
+
+Please refer to [our chart installation documentation](https://github.com/codacy/chart/blob/master/docs/install.md).
+
+Additionally, any `kubectl` command from [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md) must be executed as a `microk8s.kubectl` command.
+
+#### 1. Default Values
+
+We provide a base [values file](##values-microk8s.yaml) that you can use for your microk8s Codacy installation. This file provisions a smaller number of component replicas with limited resources (i.e. CPU and memory). Should you want to use this as a template, please pass your `values-microk8s.yaml` file on (4.) of [our chart installation](https://github.com/codacy/chart/blob/master/docs/install.md). Lastly, it is important to correctly set your `codacy-api.ingress.hosts.host` if you are using this file as a template.
```

Comment:
This is a bit weird, we refer to the chart installation documentation and then we say we should use `values-microk8s.yaml`, we should mention that we need to use this one instead of `values-production.yaml`